### DISCUSSION TITLE
#916 is open for 42 months!!

### DISCUSSION THREAD

**Discussion by artkat** [microsoft/microsoft-ui-xaml#8068] [General]
When will bug #916 finally get some attention??
It's already open for 42 months...

**Comment by ruihuasun** [comment]
I really wish the winui3 team would move fast, especially now that other non-Microsoft frameworks are growing fast despite their performance issues.

**Comment by jozefizso** [comment]
Going back to Windows desktop after a year spend with React and SwiftUI the state of WinUI 3 is alarming.

The other UI frameworks just work. WinUI 3 feels like undocumented mess of MFC from 30 years ago.

**Comment by christosk92** [comment]
try x:load
